src/main/webapp/index.html

```diff
@@ -183,6 +184,18 @@ <h1><a href="/">Census Viz</a></h1>
       <a id="census-link" target="_blank" rel="noopener">Explore more data from the Census API</a>
       <br>
       <div id="data-table"></div>
+      <div id="yearly-total-chart-loading-msg"></div>
+      <div id="yearly-total-chart" class="yearly-chart"></div>
+      <div id="yearly-county-chart-loading-msg"></div>
+      <form id="county-form" onsubmit="createYearlyCountyChart(); return false;">
+        <label for="county-list">View yearly data for a county</label>
+        <input type="list" list="county" name="county" id="county-list" autocomplete="off" required
+          onfocus="storeValueAndEmpty('county')" onfocusout="replaceValueIfEmpty('county')" value="Select a county">
+        <datalist id="county"></datalist>
+        </input>
+        <input type="submit" value="Submit">
```

Comment:
Any reason why this doesn't say "Go" like our main query does?

Reply:
Nope, switched to "Go"